### DISCUSSION TITLE
download latest stable pre-compied open-watcom

### DISCUSSION THREAD

**Discussion by sagudev** [open-watcom/open-watcom-v2#380] [Ideas]
Where can I download latest stable open-watcom installer? (for Windows)

- Sourceforge has got installers from 2015
- http://www.openwatcom.org/download.php has version 1.90
There are no latest pre-compied open-watcom for other OS too.

**Comment by sezero** [comment]
At present, there are no newer official releases than 1.9.0.
Your options are:

- snapshot builds from the official OW perforce repository, includes installers:
  https://efbe.musca.uberspace.de/

- snapshot builds from this fork of OW (no installers):
  https://github.com/open-watcom/travis-ci-ow-builds/archive/master.zip
  https://github.com/open-watcom/travis-ci-ow-builds

**Comment by sagudev** [comment]
maybe it will be good to add 

> snapshot builds from the official OW perforce repository, includes installers:
> https://efbe.musca.uberspace.de/
to README.md file so evryone could find latest installers.

And maybe add some tags.

**Comment by jmalak** [comment]
First, this project is fork of OW 1.9 to port OW to 64-bit, and do next development, because official development is dead. 
Therefore installers from OW 1.9 have no sense for this project.
We can regularly create installers but only for 32-bit version of OW.
64-bit version you must install manualy,
Anyway manual installation is simple see [readme](https://github.com/open-watcom/travis-ci-ow-builds)

**Comment by ArmstrongJ** [comment]
Just a reminder that nightly builds in simple zip archives are also available at:

http://buildbot.approximatrix.com/releases/

**Comment by sagudev** [comment]
Ok I understand OW 1.9 is not this github project, but still why the project doesn't have download links for installers at least in the [main site](http://open-watcom.github.io/open-watcom/)

**Comment by XVilka** [comment]
To be honest would be nice to have git tag for sources as well, for packaging in Linux/BSD distributions.

**Comment by jmalak** [comment]
@sagudev  the reason is my limited free time.
Installers has low priority on my to do list, because I am concentrated now to more important C++ compiler and it's regression tests to fix some problem with 64-bit version of C++ compiler.
If you are interested in creation of installers regular way in Travis build then you can help.
All the necessary staff exist in OW, only need to migrate appropriate commands from OW build server to script for new stage in Travis build process and ensure back transfer of build installers to GitHub repository.
@XVilka I am not sure if it has sense. Normaly anybody can download sources from GitHub.
If you mean to use GNU Linux build system then It is useless, because OW has it's own multi-platform build system with full cross-compile support. You can build OW on any supported OS and build for any supported host. You can build on OS2, Linux, Windows and DOS full OW for all targets without any special configuration/installation. This is not possible on GNU Linux centric build system.

**Comment by jmalak** [comment]
Anyway manual installation on any OS is simple. Only a few environment variables is setup to follow installation directory. Take into account OW is multi platform and works on various OS-es that can not depend on any OS specific features. I understand that if exist package for Linux/BSD it can be installed by regular way, but it is too specific to implement it for very small gain.
If you are interesting in this stuff then You are welcome to work on it. I can help you with OW details if you will need.

**Comment by revelator** [comment]
i sometimes create new installers from checkout which i then host on my sourceforge site.
They are not allways up to date because of real life time constraints, but i do try to make the most stable builds avaliable there. Sometimes im also required to modify the installer scripts because of new tools or removal of deprecated functions which again means that the installers are not in total sync with the source tree.

**Comment by jmalak** [comment]
I do time-to-time full OW build including installers and correct files for distribution if necessary.
Mostly file.dat is changed to reflect these change if something is add or removed.
Best is to do OW build from clean state, after run clean script and after full OW build create installers.
64-bit installers required merge of OW build from 64-bit hosts and run installers build on merged build.

**Comment by jmalak** [comment]
OW installers are build regulary now.
https://github.com/open-watcom/open-watcom-v2/releases
